iina/Preference.swift

```diff
@@ -372,6 +370,24 @@ struct Preference {
     }
   }
 
+  enum HardwareDecoderOption: Int {
```

Comment:
Why not just make the enum backed by `String`? I don't see any place that the actual integer value is used.